uk.co.compucorp.civicrm.tasksassignments/views/modal/document.html

```diff
@@ -234,20 +234,20 @@ <h4 class="modal-title" ng-switch="documentModal.isRole('admin')">
               </div>
             </div>
           </div>
-          <!-- Assignment -->
+          <!-- Workflow -->
```

Comment:
We do not store comments in the HTML code, do we?

Reply:
no, but given how big and complex this file is (it has something like 10 levels of indentation), i thought that i could keep this legacy stuff until finally T&A is overhauled

Reply:
Ok